packages/apollo-link-retry/src/retryLink.ts

```diff
@@ -148,7 +148,21 @@ class RetryableOperation<TValue = any> {
 
   private onError = async error => {
     this.retryCount += 1;
-
+    // Retry if the server tells us to.
+    if (
+      error &&
+      error.response &&
+      error.response.status === 429 &&
+      error.response.headers['Retry-After']
+    ) {
+      this.scheduleRetry(
```

Comment:
Doesn't this end up scheduling two retries if the response is a 429 (one in the `if` block, and one further down)?